include/ck_tile/ops/fmha/pipeline/block_fmha_pipeline_qr_ks_vs.hpp

```diff
@@ -58,6 +58,19 @@ struct BlockFmhaPipelineQRKSVS
     static constexpr bool kStoreLSE         = Problem::kStoreLSE;
     static constexpr bool kHasDropout       = Problem::kHasDropout;
 
+    using Gemm_0 = remove_cvref_t<decltype(Policy::template GetQKBlockGemm<Problem>())>;
+    static constexpr auto WG0_config = Gemm_0::Policy::template GetWarpGemmMWarpNWarp<Problem>();
+    using WG0                        = remove_cvref_t<decltype(WG0_config.template at<0>())>;
+    static constexpr index_t MWarp0  = WG0_config.template at<1>();
+    static constexpr index_t NWarp0  = WG0_config.template at<2>();
+    static constexpr index_t WG0_kM  = WG0 ::WarpGemmAttribute::Impl::kM;
+    static constexpr index_t WG0_kN  = WG0 ::WarpGemmAttribute::Impl::kN;
+    static constexpr index_t WG0_kK  = WG0 ::WarpGemmAttribute::Impl::kK;
+    static constexpr uint32_t DS_READ_BARRIER = 0x100; // Barrier for DS (data share) read
```

Comment:
suggrest to rename DS_READ_BARRIER as SCHD_MARK_DS_READ to align with the enum class definition [here](https://github.com/llvm/llvm-project/blob/main/llvm/lib/Target/AMDGPU/AMDGPUIGroupLP.cpp#L65-L81)